schema/bom-1.6.schema.json

```diff
@@ -3470,6 +3475,215 @@
       "title": "Data Classification",
       "description": "Data classification tags data according to its type, sensitivity, and value if altered, stolen, or destroyed."
     },
+    "environmentalConsiderations": {
+      "type": "object",
+      "title": "Environmental Considerations",
+      "description": "Describes various environmental impact metrics.",
+      "additionalProperties": false,
+      "properties": {
+        "energyConsumptions": {
+          "title": "Energy Consumptions",
+          "description": "Describes energy consumption information incurred for one or more component lifecycle activities.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/energyConsumption"
+          }
+        }
+      }
+    },
+    "energyConsumption": {
```

Comment:
the values of `activity` are very ML-specific.
therefore, i think a name prefix would be suitable. like `mlEnergyConsumption`

Reply:
I agree that these values are ml-specific today, but to build on my thoughts above, other CI-CD related activities could be added in the future and then backing off the "ml" name would be more challenging.

Reply:
This is not a blocking issues IMO.

Reply:
done